src/main/java/duke/Ui.java

```diff
@@ -0,0 +1,91 @@
+package duke;
+
+import java.util.Scanner;
+
+/** Handles input/output for Duke. */
+public class Ui {
+
+    private Scanner scanner;
+    private StringBuilder msg;
```

Comment:
Another small detail, maybe it could be better to use the full word for the variable?